dashboard/config/routes.rb

```diff
@@ -395,6 +395,7 @@ module OPS
         post :facilitator, to: 'facilitator_applications#create'
       end
 
+      get 'applications/quick_view/:role',  action: 'quick_view', controller: 'applications'
```

Comment:
This might be better as an additional RESTful route on the `applications` resource below, with `role` as a query param.

That would look like:

```ruby
resources :applications, controller: 'applications', only: [:index, :show, :update] do
  collection do
    get :quick_view
  end
end
```

The generated path would be `/api/v1/pd/applications/quick_view` and then you would call it with a query param like `/api/v1/pd/applications/quick_view?role=csf_facilitator`. The controller can remain the same. It doesn't care whether the param comes from query params or `:variables` in the url.